modules/videopress/class.videopress-gutenberg.php

```diff
@@ -164,6 +165,32 @@ public function render_video_block_with_videopress( $attributes, $content ) {
 			1
 		);
 	}
+
+	/**
+	 * Replaces the video uploaded in the block editor.
+	 *
+	 * Enqueues a script that registers an API fetch middleware replacing the video uploads in Gutenberg so they are
+	 * uploaded against the WP.com API media endpoint and thus transcoded by VideoPress.
+	 */
+	public function override_video_upload() {
+		// Bail if WP.com site.
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
@jeherve do we still do this? It's to be avoided at all costs in my mind (but I don't remember why).

Reply:
Yeah, that's okay. When possible we try to skip it (by using filters for example), but we still use this constant quite a bit in Jetpack.

Reply:
> When possible we try to skip it 

In that case, let's avoid it :) I added to D33658-code the filter removal @kwight developed in D33744-code?id=123028, and removed the WP.com bail in 813a173